### PR TITLE
Prepare for Polars support

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
push csv production to data accessors

### CODE REVIEW

**Review by namnguyen20999 — APPROVED** [review]
lgtm

File: taipy/gui/.gitignore [hunk 1]
```diff
@@ -56,3 +56,5 @@ gui/packaging/taipy-gui.gen.d.ts
 
 # GUI Build
 webapp
+
+!data
```
Comment (FabienLelaquais) [inline taipy/gui/.gitignore:60]: That is too broad a name at the top.
I suspect that for *your* data: *FLLdata*?
Anyway I would not commit that

**Review by FabienLelaquais — APPROVED** [review]
👍

File: taipy/gui/.gitignore (hunk 1 above)
Comment (FredLL-Avaiga) [reply taipy/gui/.gitignore:60]: I had to do it as someone committed a change to the root .ignore file with Data on it.
We don't want gui/data to be ignored!

**Pull request merged**